cray_product_catalog/catalog_update.py

```diff
@@ -50,13 +50,15 @@
 from cray_product_catalog.schema.validate import validate
 from cray_product_catalog.util.k8s import load_k8s
 from cray_product_catalog.util.merge_dict import merge_dict
+from cray_product_catalog.util.catalog_data_helper import split_catalog_data, format_product_cm_name
 
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
 # Parameters to identify config map and content in it to update
 PRODUCT = os.environ.get("PRODUCT").strip()  # required
 PRODUCT_VERSION = os.environ.get("PRODUCT_VERSION").strip()  # required
 CONFIG_MAP = os.environ.get("CONFIG_MAP", "cray-product-catalog").strip()
+PRODUCT_CONFIG_MAP = format_product_cm_name(CONFIG_MAP, PRODUCT)
```

Comment:
@lathanm , lets not env variables with other variables, its only used in main, we can call it from there , no need for global.